cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -67,20 +68,49 @@ public void export(Network network, Properties params, DataSource ds, Reporter r
         String filenameTp = baseName + "_TP.xml";
         String filenameSsh = baseName + "_SSH.xml";
         String filenameSv = baseName + "_SV.xml";
+
+        // Reference data (if required) will come from imported boundaries
+        // We may have received a sourcing actor as a parameter
+        String sourcingActorName = Parameter.readString(getFormat(), params, SOURCING_ACTOR_PARAMETER, defaultValueConfig);
+        String countryName = null;
+        if (sourcingActorName == null || sourcingActorName.isEmpty()) {
```

Comment:
What about if SourcingActor given as parameter is wrong? Do we not want to log a warning but still check if there's only one country ?

Reply:
@flo-dup confirms me that it is an override, even if the parameter is wrong.